src/main/java/com/eternitywall/ots/Timestamp.java

```diff
@@ -242,7 +234,7 @@ public TimeAttestation shrink() throws Exception {
         for (Iterator<Entry<Op, Timestamp>> it = this.ops.entrySet().iterator(); it.hasNext(); ) {
             Map.Entry<Op, Timestamp> entry = it.next();
             Timestamp timestamp = entry.getValue();
-            Op op = entry.getKey();
+            Op op = entry.getKey();    // TODO: Never used...
```

Comment:
you can just remove it

Reply:
But this PR is ONLY comments and JavaDoc. I wanted to make this PR as simple as possible for you to accept. I will remove it in the next next PR, which you haven't seen yet. I wrote that code two days ago but haven't submitted it as a PR yet.

Reply:
I removed this TODO also, will remember to fix this later.